### ISSUE TITLE
Gracefully handle failure to retrieve an STH or consistency proof

### ISSUE THREAD

**Issue by AGWA** [SSLMate/certspotter#9]
The Rocketeer log is distributed and is frequently unable to return a consistency proof.  certspotter should only raise an error if this happens on several subsequent certspotter runs.

**Comment by gene1wood** [comment]
Here's an example of the error message that's returned, making the email feed noisy
```
/path/to/certspotter: https://ct.ws.symantec.com: 2016/08/10 20:39:05 Error retrieving STH from log: Get https://ct.ws.symantec.com/ct/v1/get-sth: net/http: request canceled
```

**Comment by AGWA** [comment]
Another common error message, for reference:

```
certspotter: https://ct.googleapis.com/rocketeer: 2016/08/09 16:42:17 Error fetching consistency proof: GET https://ct.googleapis.com/rocketeer/ct/v1/get-sth-consistency?first=23041084&second=23073883: 400 Bad Request ({
  "error_message": "Second tree size (23073883) cannot be greater than current tree size (23057417)",
  "success": false
}
)
```

**Comment by AGWA** [comment]
Would be solved by #63.
